### ISSUE TITLE
Media Library: Fix nested button elements

### ISSUE THREAD

**Issue by swissspidy** [GoogleForCreators/web-stories-wp#4874]
## Bug Description

There's a React warning in dev mode due to nested `<button>` elements in the media library.

```
edit-story.js:154794 Warning: validateDOMNesting(...): <button> cannot appear as a descendant of <button>.
    in button (created by button__StyledButton)
    in button__StyledButton (at button/index.js:210)
    in ForwardRef(More) (created by dropDownMenu__MoreButton)
    in dropDownMenu__MoreButton (at dropDownMenu.js:110)
    in div (at dropDownMenu.js:107)
    in DropDownMenu (at mediaElement.js:273)
    in div (created by mediaElement__InnerContainer)
    in mediaElement__InnerContainer (at mediaElement.js:249)
    in button (created by mediaElement__Container)
    in mediaElement__Container (at mediaElement.js:235)
    in MediaElement
    in MediaElement (at mediaGallery.js:49)
```

## Expected Behaviour

No accessibility issues due to nested button elements

## Additional Context

- Plugin Version: 1.0.0
- Operating System: macOS
- Browser: Chrome 86

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by swissspidy** [comment]
IIRC this was fixed recently
